requirements/AUR_RT-0040/reset-reason/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+echo -e "Building binary\n"
+make leon3 debug
+echo -e "\nExecuting binary\n"
+sis-2.29 -leon3 -dumbio -uart1 system.log -rt work/binaries/leon3 < gdb.batch
+echo -e "\nTest result\n"
+cat system.log
```

Comment:
Maybe we can place this script in Makefile, one file less

Reply:
Makes sense, will do.